features/admin.sms-templates.v1/api/use-get-sms-template.ts

```diff
@@ -51,7 +51,7 @@ const useGetSmsTemplate = <Data = SMSTemplate, Error = RequestErrorInterface>(
         method: HttpMethods.GET,
         url:
             store.getState().config.endpoints.smsTemplates +
-            `/template-types/${templateType}/org-templates/${smsLocale}`
+            `/template-types/${templateType}/org-templates/${smsLocale}?resolve=true`
```

Comment:
What is the purpose of passing this param?

Reply:
https://github.com/wso2-enterprise/asgardeo-product/issues/28008

Reply:
When SMS templates are viewed in sub organizations via the console application, admin can't see the inherited SMS template.